lib/src/dart3_suggestors/null_safety_prep/required_flux_props.dart

```diff
@@ -77,7 +86,15 @@ class RequiredFluxProps extends RecursiveAstVisitor with ClassSuggestor {
   }
 
   void yieldNewCascadeSection(CascadeExpression node, String newSection) {
-    final offset = context.sourceFile.getOffsetOfLineAfter(node.target.offset);
+    final sf = context.sourceFile;
+    final targetLineOffset = sf.getOffset(sf.getLine(node.target.offset));
+    int offset;
+    if (targetLineOffset == sf.getOffset(sf.getLine(node.target.end))) {
+      // Cascade on a single line / same line as the target, add the new setter(s) before the semicolon
+      offset = node.target.end;
+    } else {
+      offset = sf.getOffsetOfLineAfter(node.target.offset);
+    }
```

Comment:
#nit why are we trying to conditionally insert it on a new line? This will all end up getting formatted anyways
```suggestion
    final offset = node.target.end;
```

Reply:
Addressed in https://github.com/Workiva/over_react_codemod/pull/268/commits/d36a65610c17e2d5768b344bc5ca0038efe5206d